samples/open-telemetry/otel.yaml

```diff
@@ -60,6 +60,9 @@ spec:
     type: RollingUpdate
   template:
     metadata:
+      annotations:
```

Comment:
should make it a label - label si preferred to annotation since it skips the webhook entirely (vs applying the webhook, but a NOP)

Reply:
updated to use label